edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml

```diff
@@ -1,264 +1,293 @@
-
-<project name="athena-project-build" default="deploy">
-  <property file="${user.home}/wpilib/cpp/${version}/ant/ni_image.properties"/>
-
+<?xml version="1.0" encoding="utf-8"?>
+<project name="athena-project-build"
+         default="deploy">
+  <property file="${user.home}/wpilib/cpp/${version}/ant/ni_image.properties" />
   <!-- Load Tasks -->
   <taskdef resource="net/sf/antcontrib/antlib.xml">
     <classpath>
-      <pathelement location="${wpilib.ant.dir}/ant-contrib.jar"/>
+      <pathelement location="${wpilib.ant.dir}/ant-contrib.jar" />
     </classpath>
   </taskdef>
-  <taskdef resource="net/jtools/classloadertask/antlib.xml" classpath="${classloadertask.jar}"/>
-  <classloader loader="system" classpath="${jsch.jar}"/>
-  <taskdef name="webdav" classname="com.github.sardine.ant.SardineTask">
+  <taskdef resource="net/jtools/classloadertask/antlib.xml"
+           classpath="${classloadertask.jar}" />
+  <classloader loader="system"
+               classpath="${jsch.jar}" />
+  <taskdef name="webdav"
+           classname="com.github.sardine.ant.SardineTask">
     <classpath>
-      <pathelement location="${wpilib.ant.dir}/sardine-5.8-SNAPSHOT.jar"/>
-      <pathelement location="${wpilib.ant.dir}/httpclient-4.5.3.jar"/>
-      <pathelement location="${wpilib.ant.dir}/httpcore-4.4.6.jar"/>
-      <pathelement location="${wpilib.ant.dir}/commons-codec-1.10.jar"/>
-      <pathelement location="${wpilib.ant.dir}/commons-logging-1.2.jar"/>
+      <pathelement location="${wpilib.ant.dir}/sardine-5.8-SNAPSHOT.jar" />
+      <pathelement location="${wpilib.ant.dir}/httpclient-4.5.3.jar" />
+      <pathelement location="${wpilib.ant.dir}/httpcore-4.4.6.jar" />
+      <pathelement location="${wpilib.ant.dir}/commons-codec-1.10.jar" />
+      <pathelement location="${wpilib.ant.dir}/commons-logging-1.2.jar" />
     </classpath>
   </taskdef>
   <typedef resource="edu/wpi/first/ant/roboriotasks.xml">
     <classpath>
-      <pathelement location="${wpilib.ant.dir}/roborio-ant-tasks-1.0.jar"/>
-      <pathelement location="${wpilib.ant.dir}/httpclient-4.5.3.jar"/>
-      <pathelement location="${wpilib.ant.dir}/httpcore-4.4.6.jar"/>
-      <pathelement location="${wpilib.ant.dir}/commons-codec-1.10.jar"/>
-      <pathelement location="${wpilib.ant.dir}/commons-logging-1.2.jar"/>
+      <pathelement location="${wpilib.ant.dir}/roborio-ant-tasks-1.0.jar" />
+      <pathelement location="${wpilib.ant.dir}/httpclient-4.5.3.jar" />
+      <pathelement location="${wpilib.ant.dir}/httpcore-4.4.6.jar" />
+      <pathelement location="${wpilib.ant.dir}/commons-codec-1.10.jar" />
+      <pathelement location="${wpilib.ant.dir}/commons-logging-1.2.jar" />
     </classpath>
   </typedef>
-
-  <target name="clean" description="Clean up all build and distribution artifacts.">
-    <delete dir="${build.dir}"/>
-    <delete dir="${dist.dir}"/>
+  <target name="clean"
+          description="Clean up all build and distribution artifacts.">
+          
```

Comment:
No, but I don't know if I care.  I much prefer automatically formatted even if it has a few inconsistencies.